Sources/SmithyTelemetry/Context/TelemetryContext.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Presuming these `Sources/SmithyTelemetry` files are all largely identical to the ones that were in `ClientRuntime`?